assets/js/blocks/cart-checkout/cart-i2/frontend.js

```diff
@@ -45,30 +47,36 @@ const CartFrontend = ( props ) => {
 
 const getProps = ( el ) => {
 	return {
-		emptyCart: el.innerHTML,
-		attributes: getValidBlockAttributes( blockAttributes, el.dataset ),
+		attributes: getValidBlockAttributes(
+			blockAttributes,
+			!! el ? el.dataset : {}
+		),
 	};
 };
 
-const getErrorBoundaryProps = () => {
-	return {
-		header: __( 'Something went wrong…', 'woo-gutenberg-products-block' ),
-		text: __(
-			'The cart has encountered an unexpected error. If the error persists, please get in touch with us for help.',
-			'woo-gutenberg-products-block'
-		),
-		showErrorMessage: CURRENT_USER_IS_ADMIN,
-		button: (
-			<button className="wc-block-button" onClick={ reloadPage }>
-				{ __( 'Reload the page', 'woo-gutenberg-products-block' ) }
-			</button>
-		),
-	};
+const Wrapper = ( { children } ) => {
+	// we need to pluck out receiveCart.
+	// eslint-disable-next-line no-unused-vars
+	const { extensions, receiveCart, ...cart } = useStoreCart();
```

Comment:
I omitted `useCheckoutExtensionData` from here, it didn't make sense. I might omit `useValidation` as well.